python/pip_install/extract_wheels/bazel.py

```diff
@@ -247,7 +247,7 @@ def entry_point(pkg, script = None):
         def install_deps():
             fail("install_deps() only works if you are creating an incremental repo. Did you mean to use pip_parse()?")
         """.format(
-            repo=repo_name,
+            repo="@%s" % repo_name.removeprefix("@rules_python.pip."),
```

Comment:
This is a bit awkward. The repository's `name` resolves to the bzlmod hierarchichal name which is not the label that a downstream user should use.